src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -169,7 +175,7 @@ public DynamoAnalyticsClient(DynamoModel dynamoModel)
             //Dynamo app version.
             var appversion = dynamoModel.AppVersion;
 
-            product = new ProductInfo() { Name = "Dynamo", VersionString = appversion, AppVersion = dynamoModel.Version };
+            product = new ProductInfo() { Id = "DYN", Name = "Dynamo", VersionString = appversion, AppVersion = dynamoModel.Version, BuildId = "CB159AA3-2F20-4791-AE8C-8939E90C9600", ReleaseId = "2.5.0", MasterId = "{4C7B248C-56D3-488F-8E23-8CAFD2651968}" };
```

Comment:
whats the process around these ids and when we increment them?
Latest released version is 2.5.3

Reply:
Again...this is test dummy data.
From my discussion with other teams....it seems that the proper releaseId format is the targeted release year (ex. 2021) So we would need to change the UPI configs
THe rest of the identifiers (from my understanding) do not play any real roles (they should just reflect the configs in UPI..and they do not need to change from release to release)

Basically it sounds like the ReleaseId is the only config that needs to change from release to release.

Reply:
@mjkkirschner We update them everytime we RTM. They need to be first populated from UPI and then reflected in Code here to be passed in. These will need to be documented when this gets in.